### PR TITLE
update to 1.20.3/4

### PR DESCRIPTION
**Pull request by MeeniMc**
fixes: #309

### CODE REVIEW

**Comment by MeeniMc** [comment]
fixes: #309

**Comment by MeeniMc** [comment]
a built binary is available from https://github.com/MeeniMc/carpet-extra/releases/tag/v1.4.128-meenimc.1

Please report bugs and issues with this release in this PR, and not in the issue tracker, as this is not an officially supported release.

File: gradle.properties [hunk 1]
```diff
@@ -21,7 +21,7 @@ org.gradle.jvmargs=-Xmx1G
 	# The Curseforge versions "names" or ids for the main branch (comma separated: 1.16.4,1.16.5)
 	# This is needed because CF uses too vague names for prereleases and release candidates
 	# Can also be the version ID directly coming from https://minecraft.curseforge.com/api/game/versions?token=[API_TOKEN]
-	release-curse-versions = Minecraft 1.20:1.20.2
+	release-curse-versions = Minecraft 1.20.3:1.20.4
```
Comment (altrisi) [inline gradle.properties:24]: Oops, didn't notice this isn't correct (it follows some weird curseforge versioning where the prefix is the minor only and suffix is with patch), easy fix though

**Comment by AnonymousPuffin** [comment]
Currently running Fabric 1.20.4 server with Carpet and Carpet Extra v1.4.128 with emptyShulkerBoxStackAlways set to true, but empty Shulkers are not stacking. The mods are installed on both the client and the server.

Any assistance would be greatly appreciated.

**Pull request merged**